src/plugin/vars/queueDelaySeconds.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "Queue Delay Seconds.",
+  "details": [
+    "The length of time a queue will postpone the delivery of new messages to a consumer."
+  ],
+  "type": "integer"
+}
```

Comment:
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-delay-queues.html

Reply:
can we put a default here?